marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-temporal-user.json

```diff
@@ -0,0 +1,16 @@
+{
+  "role-name": "data-hub-temporal-user",
+  "description": "Primarily intended to support temporal operations.",
```

Comment:
For consistency with other "data-hub-" roles, let's say: "Permits temporal operations on documents"

Reply:
Changed